pom.xml

```diff
@@ -834,7 +835,11 @@
                 <artifactId>mysql-connector-java</artifactId>
                 <version>${mysql-connector-java.version}</version>
             </dependency>
-            
+            <dependency>
```

Comment:
Please do not depend other driver. Please add the driver  to classpath by yourself